src/api/search/src/bin/validation.js

```diff
@@ -75,14 +75,30 @@ const advancedQueryValidationRules = [
     .bail(),
 ];
 
+const getValidationRules = (searchType) => {
+  let rules;
+  switch (searchType) {
+    case '/':
+      rules = queryValidationRules;
+      break;
+    case '/advanced':
+      rules = advancedQueryValidationRules;
+      break;
+
+    default:
+      break;
```

Comment:
I don't like that we can fall through this and get to a situation where `rules` is undefined.  What is the reason for not using a ternary or if/else?

Reply:
We could you're right. I can revert it back and use route.path - should work the same. I got carried away with the previous solution.

Reply:
Can I suggest that if we have no users of the advanced route, we don't do this at all until we need it?

Let's learn from our time on this, and prioritize getting the tests landed first, then add the implementation later.

Reply:
i'm ok with that, however the front end UI will need this. I guess I can put both in together then?